aklite-callback-example/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.12
+
+COPY example-callback.sh /
+COPY handle-callback-example.sh /
+
+ENTRYPOINT ["/handle-callback-example.sh"]
```

Comment:
I suppose this is probably how we should show for an example. However, you could just bind mount these files in the docker-compose.yml and not have to bother with building a custom container - just use alpine:3.12